### ISSUE TITLE
Global CONTRIBUTING.md

### ISSUE THREAD

**Issue by glasnt** [pybee/pybee.github.io#2]
(Putting this issue under the pybee.github.io repo, as it seems like the best place to put global matters)

GitHub does fancy things with a CONTRIBUTING{,.md} file, where it will display [a link to it on new issues opened](https://github.com/blog/1184-contributing-guidelines)[0]. [Example link](https://github.com/glasnt/octohat/issues/new)

The index.html on this repo already points to the [Python Community Code of Conduct](https://www.python.org/psf/codeofconduct/). This is awesome! But it would be great if this could be linked in all repos. Unfortunately, that requires creating a CONTRIBUTING file per repo (and it may or may not allow a CONTRIBUTING.rst, unfortunately.) 

My thought: create a global CONTRIBUTING under this repo. Link out to the CoC, detail all the information about how issues are welcomed. For any other repos, create a CONTRIBUTING stub file, with just a basic link out to the main thing. Something like: 

```
# CONTRIBUTING

PyBee operates under a Code of Conduct. See [CONTRIBUTING to PyBee](https://github.com/pybee/pybee.github.io)

```

The canonical CONTRIBUTING file should probably also describe how much contributions are welcomed, even more than the current stub does (currently for VOC: "If you experience problems with VOC, log them on GitHub. If you want to contribute code, please fork the code and submit a pull request.")

Pro: one canonical version of CONTRIBUTING linked to all repos. Stub on all repos click through to the canonical version
Pro: linking to a canonical CONTRIBUTING allows easier communication of any changes (e.g. if a different CoC is chosen later)
Con: all repos need a CONTRIBUTING file to appear in the GitHub Issues flash message. Mitigation - create a sufficiently short and sweet mini-CONTRIBUTING file to link to the canonical version. 
Con: using the github pages repo to store this uses an existing repo, but I think creating a new repo just for this kind of documentation is overkill

More than happy to discuss this further. 

[0] This, and many other awesome tips, from @pjf's [FOSS Community](https://2015.osdc.com.au/schedule/presentation/13/) talk at [OSDC](https://osdc.com.au) [(video)](https://www.youtube.com/watch?v=dkMFoo60d0s)

**Comment by freakboy3742** [comment]
Broadly speaking - sounds like a great idea - +1.

The "canonical CONTRIBUTING in pybee.github.io; stub in all projects" sounds like a reasonable approach, too. 

My biggest concern is about the content on the canonical CONTRIBUTING file, and the extent to which having a long set of contribution guidelines/instructions might be intimidating in itself. I suspect this might come down to doing some good copyediting so that the lede isn't buried, but the details are there if you want them.

PJF's talk is on my to-watch list.

**Comment by freakboy3742** [comment]
Resolved by #3
